go.mod

```diff
@@ -13,11 +13,11 @@ require (
 	golang.org/x/tools v0.34.0
```

Comment:
There are some more occurences of v0.34.0-alpha.2 (in another go.mod file though)